apps/modernization-ui/src/apps/patient/add/AddPatient.tsx

```diff
@@ -143,6 +144,12 @@ const AddPatient = () => {
         });
     };
 
+    const navigate = useNavigate();
+
+    const handleCancel = () => {
+        navigate(-1);
```

Comment:
issue:  this needs to use `useSearchNavigation` to navigate to Patient Search.  This page can be navigated to directly which makes the cancel button not return to patient search.